plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -95,6 +96,35 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    @Test
+    public void testDatabaseMetadataSearchEscapedWildCardCharacters()
+    {
+        // wildcard characters on schema name
+        assertQuerySucceeds("SHOW TABLES FROM public");
+        assertQueryFails("SHOW TABLES FROM \"publi_\"", ".*Schema 'publi_' does not exist");
+        assertQueryFails("SHOW TABLES FROM \"pu%lic\"", ".*Schema 'pu%lic' does not exist");
+
+        String normalTableName = "testxsearch" + randomNameSuffix();
+        String underscoreTableName = "\"" + "test_search" + randomNameSuffix() + "\"";
+        String percentTableName = "\"" + "test%search" + randomNameSuffix() + "\"";
+        try {
+            assertUpdate("CREATE TABLE " + normalTableName + "(a int, b int, c int) WITH (primary_key = ARRAY['a'])");
+            assertUpdate("CREATE TABLE " + underscoreTableName + "(a int, b int, c int) WITH (primary_key = ARRAY['b'])");
+            assertUpdate("CREATE TABLE " + percentTableName + " (a int, b int, c int) WITH (primary_key = ARRAY['c'])");
+
+            // wildcard characters on table name
+            assertQuerySucceeds("SHOW CREATE TABLE " + underscoreTableName);
+            assertQuerySucceeds("SHOW CREATE TABLE " + percentTableName);
```

Comment:
Please verify if the result contains the expected `primary_key` table property in 3 tables. e.g. 
```java
assertThat((String) computeScalar("SHOW CREATE TABLE " + x)).contains("primary_key = ARRAY['a'])");
```